add-on/src/popup/browser-action/gateway-status.js

```diff
@@ -17,12 +17,12 @@ function statusEntry ({ label, labelLegend, value, check, itemClass = '', valueC
     `
 }
 
-module.exports = function gatewayStatus ({
-  ipfsApiUrl,
+export default function gatewayStatus ({
   gatewayAddress,
   gatewayVersion,
-  swarmPeers,
-  ipfsNodeType
+  ipfsApiUrl,
+  ipfsNodeType,
+  swarmPeers
```

Comment:
if you're alphabetizing things like this manually, stop and wait for eslint automation. I prefer organized keys and imports as well, but they will quickly get out of date without a 'fixable' eslint rule.